lib/index.js

```diff
@@ -67,30 +78,44 @@ module.exports = class Breakpoints {
    * Receives the userConfig
    */
   #getUserConfig() {
-    const userConfigPath = path.resolve(process.cwd(), this.#userConfigName);
+    const userConfigPath = path.resolve(process.cwd(), this.#configName);
     try {
-      this.#userConfig = deepmerge(
-        this.#userConfig,
+      this.#config = deepmerge(
+        this.#defaultConfig,
         Breakpoints.readYamlFile(userConfigPath)
       );
-      if (!this.#userConfig.css?.enabled && !this.#userConfig.js?.enabled) {
-        console.warn(MESSAGES.noOutput(this.#userConfig));
-      }
-      if (Object.values(this.#userConfig.options).every((option) => option)) {
-        console.warn(MESSAGES.noOutput(this.#userConfig));
-      }
     } catch (error) {
       console.error(MESSAGES.userConfig(userConfigPath));
     }
   }
 
+  /**
+   * Checks if config is valid
+   *
+   * @returns {boolean}
+   */
+  #isValidUserConfig() {
+    if (
+      !this.#config.drupal?.path ||
+      (!this.#config.css?.enabled && !this.#config.js?.enabled) ||
+      !Object.values(this.#config.options).includes(true)
+    ) {
+      console.error(MESSAGES.noOutput(this.#config));
+      return false;
+    }
+    if (this.#config.drupal?.themeName === DEFAULT_CONFIG.drupal.themeName) {
```

Comment:
You removed the default theme name, so this can be shortened to `!this.#config.drupal?.themeName`.